azurerm/config.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/Azure/azure-sdk-for-go/services/automation/mgmt/2015-10-31/automation"
 	"github.com/Azure/azure-sdk-for-go/services/cdn/mgmt/2017-10-12/cdn"
 	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-06-01/compute"
-	"github.com/Azure/azure-sdk-for-go/services/containerinstance/mgmt/2018-04-01/containerinstance"
+	"github.com/Azure/azure-sdk-for-go/services/containerinstance/mgmt/2018-06-01/containerinstance"
```

Comment:
Please use the `2018-09-01` in v21.1.0

Reply:
@metacpp, per our email conversation, I will keep this one at 2018-06-01 due to significant refactor in 2018-09-01. Once the secure environment variables PR has been merged, I will create a new PR to refactor the provider to use ACI 2018-09-01.